Sweeping AI QoL

Mewchild opened the pull request:
Alright. Lots of tiny changes in this one.

- Adds a Turret control to the AI core, linked to the two turrets at the top of the stairs
- Sets Broadcast on the Public Garden Floor 3 intercomm to 0 (stops the mic from being on at round start)
- Adds several holopads across the station, improving coverage for the AI
- Adjusts the locations of/adds several cameras in public areas, removing inconvenient blind spots. These notably don't affect private spaces, such as the dorms/dorm hallways/laundry room
- Added a 5% increased chance of sexy AI interactions, across the board

Holopad coverage and strange camera blindspots were pointed out by Ech0 (Ismuth#3876).

Fixes #4595

Mewchild commented:
It seems that if you touch a number of files, and the images MapDiffBot tries to create are large, it cries and gives up due to the combined filesizes

The Camera changes were minor, and only affect areas that were in public halls, where the cams just didn't reach for no good reason. 
![image](https://user-images.githubusercontent.com/13400422/50204513-bfa0b180-032a-11e9-846a-12e093009b70.png)
![image](https://user-images.githubusercontent.com/13400422/50204518-c5969280-032a-11e9-8974-028db36c637a.png)

Holopads were added in medical, CMO Office, QM office, Numerous places in the halls to fix gaps in coverage. Pads in the bottom floor atrium were rearranged to fix coverage gaps.

Very-Soft commented:
I should note that in the base of the public garden, the camera placement was intentional, so you could sneak into the space behind the scrubbers without being spotted on cameras. It's not really all that important, but I did make that place specifically to be a sort of sneaky spot.

Mewchild commented:
Removed the extra camera from Garden 1, as per @Very-Soft's Comment. Added a camera into the SM engine, to fix a critical blindspot that prevented accessing the cooling loop and APC.

SpadesNeil commented:
Mapping errors galore.
